README.md

```diff
@@ -29,27 +29,31 @@ Prior to vSphere 8 Update 1, the Supervisor Services are only available with Sup
 
 1. Log in to [support.broadcom.com](https://support.broadcom.com)
 
-2. Select `My Downloads` on the left hand side navigation.
+2. Select `Enterprise Software` on the left hand side navigation.
 
-3. Select `Free Software Downloads available HERE`, if you are looking for free services. Search `vSphere Supervisor Services`.
+<img src="main.png" width="750" height="300"  title="Home" id="home">
 
-4. Services that need entitlements can be downloaded directly from `My Downloads`. Search for `vSphere Supervisor Services`.
+3. Select `My Downloads` on the left hand side navigation.
 
-5. If you are looking to download VMware Private AI Services, go to `My Downloads` and then search the term `VMware Private AI Services`.
+4. Select `Free Software Downloads available HERE`, if you are looking for free services. Search `vSphere Supervisor Services`.
 
-6. Next navigate to the service of choice and version you are looking to install.
+5. Services that need entitlements can be downloaded directly from `My Downloads`. Search for `vSphere Supervisor Services`.
```

Comment:
Don't need to be addressed here, but should we update the table below for which services are free, VS which need entitlements?